packages/docusaurus-theme-classic/src/index.js

```diff
@@ -7,7 +7,36 @@
 
 const path = require('path');
 
+// Need to be inlined to prevent dark mode FOUC
+const noFlash = `(function() {
+  var storageKey = 'theme';
```

Comment:
It will be good to add a comment where this value in set in `useTheme` to change this value if the string there is updated? Or import the string value from `useTheme`. Same goes for 'data-theme'